src/app/qgisapp.cpp

```diff
@@ -4186,10 +4186,11 @@ void QgisApp::setupConnections()
     {
       canvas->setCanvasColor( backgroundColor );
     }
-    if ( mapOverviewCanvas() )
+    auto &&lMapOverviewCanvas = mapOverviewCanvas();
+    if ( lMapOverviewCanvas )
```

Comment:
Not sure if your script can be updated, but a much cleaner result would be:

```
   if ( auto* lMapOverviewCanvas = mapOverviewCanvas() )
   {
     ...
   }
```

is that possible?

Reply:
> is that possible?

That's the whole point of using a script :-) Done